server/app/rest_api/rest.py

```diff
@@ -49,8 +49,60 @@ def get(self):
         return make_response(jsonify({"schema": current_app.data.schema}), 200)
 
 
+class ConfigAPI(Resource):
+    @swagger.doc({
+        "summary": "Configuration information to assist in front-end adaptation"
+                   " to underlying engine, available functionality, interactive time limits, etc",
+        "tags": ["initialize"],
+        "parameters": [],
+        "responses": {
+            "200": {
+                "description": "schema",
+                "examples": {
+                    "application/json": {
+                        "config": {
+                            "features": [
+                                {"method": "POST", "path": "/cluster/", "available": False},
+                                {
+                                    "method": "POST",
+                                    "path": "/layout/obs",
+                                    "available": True,
+                                    "interactiveLimit": 10000
+                                },
+                                {"method": "POST", "path": "/layout/var", "available": False}
+
+                            ],
+                            "displayNames": {
+                                "engine": "ScanPy version 1.33",
+                                "dataset": "/home/joe/mouse/blorth.csv"
+                            },
+                        }
+                    }
+                }
+            }
+        }
+    })
+    def get(self):
+        config = {
+            "config": {
+                "features": [
+                    {"method": "POST", "path": "/cluster/", **current_app.features["cluster"]},
+                    {"method": "POST", "path": "/layout/obs", **current_app.features["layout"]["obs"]},
+                    {"method": "POST", "path": "/layout/var", **current_app.features["layout"]["var"]},
+                    {"method": "POST", "path": "/diffexp/", **current_app.features["diffexp"]},
+                ],
+                "displayNames": {
+                    "engine": f"ScanPy Engine version {current_app.data.__version__}",
```

Comment:
I think this is potentially confusing because is this the scanpy version or the scanpy cellxgene interface?

Reply:
Do you mean I should change the verbiage? Or is the concept of versioning the engine itself confusing?

Reply:
Yeah.  cellxgene scanpy engine?  I dunno.  I'm terrible with naming things. :)